content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Cyber Resilience'
+Description: 'Cyber Resilience is the ability of an organization to withstand and recover from cyber incidents.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+CatalogContent:
+  - 'paths/cyber-resilience-risk-management'
+---
+
+**Cyber Resilience** is the ability of an organization to withstand and recover from cyber incidents. It describes the ability to continue operating despite experiencing challenging cyber events such as ransomware, Distributed Denial of Service (DDoS), social engineering, and insider threats.
+
+## Cyber Resilience In-Use:
+
+Becoming cyber resilient is essential to preventing business disruptions that may result from a cybersecurity incident. In order to become cyber resilient, an organization must first consider its current state security posture. Most importantly, the so-called crown jewels (i.e. critical business/service infrastructure) must be identified, prioritized, and protected. Actions such as application control and implementing the principle of least privilege can lessen risk and increase resilience.
+
+## Ways to Achieve Cyber Resilience
+
+In Cybersecurity, some concepts that go into achieving Cyber Resiliency are:
+
```

Comment:
In addition to the edits below, remove the line breaks between each bullet item.